db/migrate/20180203144744_add_family_albums_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddFamilyAlbumsToUsers < ActiveRecord::Migration[5.1]
+  def self.up
+    change_table :users do |t|
+      t.belongs_to :family_album, foreign_key: true
```

Comment:
Why do you need this association? I would look into has many through associations user -> family -> family_album